crates/astria-sequencer/src/app.rs

```diff
@@ -206,7 +205,13 @@ impl App {
             .context("failed to call init_chain on IbcComponent")?;
 
         state_tx.apply();
-        Ok(())
+
+        let app_hash = self
+            .prepare_commit(storage)
+            .await
+            .context("failed to prepare commit")?;
+        debug!(app_hash = %telemetry::display::base64(&app_hash), "init_chain completed");
```

Comment:
i actually do call `commit` right after `init_chain` in the `init_chain` message handler in the consensus service, so the changes are committed. i'm not sure if that's necessary though

Reply:
Ah - sorry - I wasn't meaning you weren't calling `commit` right after `init_chain` in this PR.

I was more thinking of down the line if there end up being more places calling `init_chain` then it would be good to just call `commit` inside `init_chain` so it doesn't get forgotten.

Reply:
ah okay, the cometbft behaviour is that it only ever calls `init_chain` once, so this should be fine